jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -634,15 +642,37 @@ def serialize_script(script):
     else:
         return result
 
-
 def mk_multisig_script(pubs, k):
     """ Given a list of pubkeys and an integer k,
     construct a multisig script for k of N, where N is
     the length of the list `pubs`; script is returned
     as hex string.
     """
-    return serialize_script([k] + pubs + [len(pubs)]) + 'ae'
+    return serialize_script([k] + pubs + [len(pubs)]) \
+        + 'ae' #OP_CHECKMULTISIG
 
+def mk_freeze_script(pub, locktime):
+    """
+    Given a pubkey and locktime, create a script which can only be spent
+    after the locktime as passed using OP_CHECKLOCKTIMEVERIFY
+    """
+    if not isinstance(locktime, int):
+        raise TypeError("locktime must be int")
+    if not isinstance(pub, str):
+        pub = binascii.hexlify(pub).decode()
+    return (serialize_script([locktime])
+        + 'b1' #OP_CHECKLOCKTIMEVERIFY
+        + '75' #OP_DROP
+        + serialize_script([pub])
+        + 'ac' #OP_CHECKSIG
```

Comment:
Couldn't constants from jmbitcoin/btscript.py used here? (#536 deletes it, but probably it shouldn't?)

Reply:
Those constants are available in python-bitcointx, so it more replaces than deletes it. I agree in current codebase btscript.py values can/should be used.

Reply:
When I added `btscript.py` I was doing it so I could make custom scripts like these: https://github.com/AdamISZ/CoinSwapCS/blob/master/coinswap/base.py#L138-L164 ; note that I was basically copying from python-bitcoinlib (which *tx is just a fork of, it has the same stuff).

Reply:
I didn't know about btscript.py.